pkg/ansible/ansible.go

```diff
@@ -24,11 +24,19 @@ type Result struct {
 	Value    yaml.Node
 }
 
+func decryptVariable(v *yaml.Node, password string) {
+	if bytes.HasPrefix([]byte(v.Value), vaultHeader) {
+		decrypted, _ := vault.Decrypt(v.Value, password)
```

Comment:
Might be a good idea to handle the error here if the library errors for whatever reason like if the password is invalid.

Reply:
Thanks! I actually prefer it the other way by passing it up to the caller. Its the more idiomatic Go way of doing it. I'll merge for now and make some changes to save you the trouble 🙂